metricbeat/module/apache/status/status_test.go

```diff
@@ -157,7 +157,7 @@ func TestFetchTimeout(t *testing.T) {
 	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(200)
 		w.Header().Set("Content-Type", "text/plain; charset=ISO-8859-1")
-		time.Sleep(timeout)
+		time.Sleep(timeout + 5*time.Millisecond)
```

Comment:
- Why do we override the timeout here? 
- As I see the value was set to `timeout := 50 * time.Millisecond`.

Reply:
just to make sure the timeout happens, sometimes it might fail with:
```
--- FAIL: TestFetchTimeout (0.05s)
    status_test.go:228: Expected an error, had 0. []
```

Reply:
Thanks @AndersonQ for the inline comment.